smartcontactmanager/src/main/resources/application.properties

```diff
@@ -1,14 +1,13 @@
-server.port=8080
-application.security.jwt.secret-key=84bbb9eacda25cf7d1b1ac1dd1a7e0db5af77a2cafaaafcb6b03349238223c09
-application.security.jwt.expiration=86400000
-application.security.jwt.refresh-token.expiration=604800000
-spring.datasource.url=jdbc:postgresql://localhost:5432/smart_contact_manager
-spring.datasource.username=postgres
-spring.datasource.password=password
 
-spring.jpa.hibernate.ddl-auto=create-drop
+server.port=8082
+
+spring.jpa.database=POSTGRESQL
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username=postgres
+spring.datasource.password=postgres
 spring.jpa.show-sql=true
-logging.level.org.springframework.web.servlet.handler.HandlerMappingIntrospector=ERROR
```

Comment:
Line 11 to14 are needed. You did a mistake during merging from master

Reply:
Resolved